seonshine_frontend/src/pages/signUp/ChooseUserType/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import FoodBankIcon from '@mui/icons-material/FoodBank';
+import { Box, Button, Stack, Typography, useMediaQuery } from '@mui/material';
+
+import { EnterUserTypePropsType } from '../types';
+
+const ChooseUserType = ({ handleSubmitUserType }: EnterUserTypePropsType) => {
+  const isMobile = useMediaQuery('(max-width:600px)');
```

Comment:
define util or custom hook for check device type. And move 600px for define breakpoint constant